src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -67,4 +86,91 @@ export class DashboardComponent implements OnInit, OnDestroy {
         version[0] + ' ' + version.slice(2, version.length).join(' ');
     });
   }
+
+  getPrometheusData(selectedTime: any) {
+    this.unsubClusterUtilizationServices();
+
+    this.usedCapacitySubs = this.source.subscribe(() => {
+      this.prometheusService
+        .getPrometheusData({
+          params: this.promqls[0],
+          start: selectedTime['start'],
+          end: selectedTime['end'],
+          step: selectedTime['step']
+        })
+        .subscribe((data: any) => {
+          if (data.result.length) {
+            this.usedCapacity$ = data.result[0].values;
+          }
+        });
+    });
+    this.iopsSubs = this.source.subscribe(() => {
+      this.prometheusService
+        .getPrometheusData({
+          params: this.promqls[1],
+          start: selectedTime['start'],
+          end: selectedTime['end'],
```

Comment:
Why not having only one timer for all prometheus calls?
```suggestion
    timer(0, this.interval).subscribe(() => {
      this.prometheusService
        .getPrometheusData({
          params: this.promqls[0],
          start: selectedTime['start'],
          end: selectedTime['end'],
          step: selectedTime['step']
        })
        .subscribe((data: any) => {
          if (data.result.length) {
            this.usedCapacity$ = data.result[0].values;
          }
        });
         this.prometheusService
        .getPrometheusData({
          params: this.promqls[1],
          start: selectedTime['start'],
          end: selectedTime['end'],
    });
  
    
```